apps/sms/views/shared/js/shim_host_bootstrap.js

```diff
@@ -7,6 +8,12 @@
 'use strict';
 
 exports.bootstrap = function(appInstanceId) {
+  // We can't handle system messages inside iframe right now, so we access
+  // parent window here. This should be removed once bug 818000 is resolved.
+  ActivityShim.init(
```

Comment:
Well, I'll remeasure today to have numbers, but AFAIK the initial reason to keep this service here was that we didn't have shim host iframe (it was first service we landed) :) But even without numbers I think it's good to get rid of single shim in window for the consistency and remove both bigger bridge.js and bridge service mixin from view. And once system messages mechanism is fixed, it will require single line fix here :)